zkvm/tests/zkvm.rs

```diff
@@ -239,7 +271,7 @@ fn spend_1_2() {
         predicates[2].clone(), // output 2 predicate
     );
 
-    match test_helper(correct_program) {
+    match test_helper(correct_program, &scalars) {
```

Comment:
I don't feel good about passing all these scalars around — there should be a cleaner way. But maybe we'll have a better way closer to the "signer API" a bit later, so let's just keep it explicit for now.

Reply:
for sure — it's not a very clean solution to providing secret signing information indirectly, but refactoring this with the signer API makes sense to me!